cityofaustin/janis
# 4284 Janis Search: site index generation on build

ZenHub Issue: https://app.zenhub.com/workspaces/techstack-5a78b88e1ce69f3510b678ef/issues/cityofaustin/techstack/4284

Deployed:   https://janis-v3-build-search.netlify.com/en/search

NOTE: this PR doesn't add any UI...

# Description

This is just the search index generation on build part of search. I've noted out a few files that are mostly just placeholders for the UX/UI part of search. So, it's "just-the-data" here...
- Primary functionality lives in `static.config.js`

# Types of changes


# How can this be tested?

Visit https://janis-v3-build-search.netlify.app/en/search/ to see all *un-styled* pages of staging.
- Check the links! 
- Visit: https://janis-v3-build-search.netlify.app/en/search/ to find all live pages are in the search Index.

# Checklist:
